sql/src/main/java/org/elasticsearch/action/bulk/BulkShardProcessor.java

```diff
@@ -225,6 +225,7 @@ private ShardId shardId(String indexName, String id, @Nullable String routing) {
                 routing
             ).shardId();
         } catch (IndexNotFoundException e) {
+            LOGGER.warn(String.format(Locale.ENGLISH, "Trying to insert value into unavailable partition: %s", indexName));
```

Comment:
@mfussenegger maybe the message needs to be improved. However, this warning should also occur if you insert into a closed partition - `autoCreateIndices = true` if this happens.